app/src/format.py

```diff
@@ -26,7 +26,7 @@ def format_guru_cards(
     subsections: Sequence[Subsection],
     raw_response: str,
 ) -> str:
-    response_with_citations = reify_citations(raw_response, subsections)
+    response_with_citations = reify_citations(raw_response, subsections, data_source="Guru")
```

Comment:
I'll replace all these `data_source` parameters with a more object-oriented refactoring in a separate PR.